shared/fa2_modules/test/simple_admin_wrapper.mligo

```diff
@@ -0,0 +1,34 @@
+(*
+  Wrapper around simple_admin for testing:
+  embeds simple_admin.mligo and includes `unit` entrypoints
+  that trigger `fail_if_not_admin`, `fail_if_paused`
+*)
+
+#include "../simple_admin.mligo"
```

Comment:
We have new implementation for various admin modules in `minter_sdk` repo which are not migrated to `smart_contracts` repo yet. 
Consider writing a wrapper test contract using those new modules. This way you can create a family of test wrappers for diffrent implementations of the admin module